deploy/Chart/templates/ucp/secret.yaml

```diff
@@ -0,0 +1,12 @@
+{{- if .Values.global.caCert.filename }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.global.caCert.secretName }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app.kubernetes.io/name: controller
```

Comment:
This isn't really part of the controller, I think we can take out this label.

Reply:
removed